ffbs-parker-nodeconfig/files/etc/config/parker

```diff
@@ -0,0 +1,4 @@
+
+config nodeconfig 'example'
+	option config_server 'config.community.example'
```

Comment:
I think this should be

```suggestion
config nodeconfig 'nodeconfig'
	option config_server 'config.community.example'
```

Reply:
@SmithChart still open

Reply:
See https://github.com/SmithChart/community-packages/pull/6

Reply:
@grische can you verify that this has been resolved by https://github.com/SmithChart/community-packages/pull/6 ?